doc/tutorials/using_sbl_on_up2.rst

```diff
@@ -89,51 +86,64 @@ An example of the configuration file ``uos.json``:
        "DestinationType" : "virtual",
        "PartitionLayout" : [ { "disk" : "clearlinux.img", "partition" : 1, "size" : "100M", "type" : "EFI" },
                              { "disk" : "clearlinux.img", "partition" : 2, "size" : "10G", "type" : "linux" } ],
-       "FilesystemTypes" : [ { "disk" : "clearlinux.img", "partition" : 1, "type" : "ext2" },
+       "FilesystemTypes" : [ { "disk" : "clearlinux.img", "partition" : 1, "type" : "vfat" },
                              { "disk" : "clearlinux.img", "partition" : 2, "type" : "ext4" } ],
        "PartitionMountPoints" : [ { "disk" : "clearlinux.img", "partition" : 1, "mount" : "/boot" },
            		          { "disk" : "clearlinux.img", "partition" : 2, "mount" : "/" } ],
-       "Version": 26880,
+       "Version": 27050,
        "Bundles": ["kernel-iot-lts2018", "openssh-server", "software-defined-cockpit", "os-core", "os-core-update"]
    }
 
 .. note::
    To generate the image with a specified version, please modify
    the "Version" argument, ``"Version": 26000`` instead
-   of ``"Version": 26880`` for example.
+   of ``"Version": 27050`` for example.
 
-Clone the source code of ``acrn-hypervisor`` and build SOS and LaaG image:
+Clone the source code of ``acrn-hypervisor`` for building:
 
 .. code-block:: none
+   
+   $ cd ~
+   $ git clone https://github.com/projectacrn/acrn-hypervisor
+
+Build SOS and LaaG image:
+   
+.. code-block:: none   
 
-   cd ~
-   git clone https://github.com/projectacrn/acrn-hypervisor
-   sudo ./acrn-hypervisor/devicemodel/samples/up2/create-up2-images.sh --images-type all \
-      --clearlinux-version 26880 --laag-json uos.json --acrn-code-path ~/acrn-hypervisor/
+   $ sudo -s 
+   # ./acrn-hypervisor/devicemodel/samples/up2/create-up2-images.sh --images-type all \
+      --clearlinux-version 27050 --laag-json uos.json --acrn-code-path ~/acrn-hypervisor/
```

Comment:
This is most likely broken since you are now root and `~` will therefore expand to `/root` which is not where the `acrn-hypervisor` was cloned.